e2e/test/prerequisites/readme.md

```diff
@@ -31,3 +31,17 @@ docker run -d --restart unless-stopped --name azure-iot-tinyproxy -p 127.0.0.1:8
 Alternatives:
  - TinyProxy for Linux/Mac: https://tinyproxy.github.io/
  - Squid for Windows: https://chocolatey.org/packages/squid
+
+### ASC for IoT Security Message E2E Tests
+
+Prerequisites:
+
+- The IoT hub configured in `Configuration.IoTHub` must be onboarded to ASC for IoT security, with the feature “Store raw device security events in LogAnalytics.” set to on.
```

Comment:
Please add more details on how to do this for 3rd party customers. (e.g. File a request in Azure Portal.)